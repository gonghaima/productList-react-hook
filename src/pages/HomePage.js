import React from "react";
import Pagination from "../components/Pagination";
import Header from "../components/Header";
import Main from "../components/Main";

export default () => {
  return (
    <div className="App">
      <Header />
      <Main />
      <Pagination />
    </div>
  );
};
