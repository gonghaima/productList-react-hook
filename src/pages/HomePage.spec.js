import React from "react";
import { shallow } from "enzyme";
import HomePage from "./HomePage";

it("HomePage component renders without crashing", () => {
  const homePageComponent = shallow(<HomePage />);
  expect(homePageComponent).toMatchSnapshot();
});
