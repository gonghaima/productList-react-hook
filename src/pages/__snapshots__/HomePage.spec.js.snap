// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component renders without crashing 1`] = `ShallowWrapper {}`;

exports[`renders without crashing 1`] = `ShallowWrapper {}`;
